testsuite-native-image/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2017 The Netty Project
```

Comment:
Probably better dropping this header as well because we don't want to copyright a built script.

Reply:
Ah got it. Let's leave it as it us.